### PR TITLE
fix - detect_async method and IGNORE zenguard e2e tests

### PR DESCRIPTION
**Pull request by nurvdil**

### CODE REVIEW

**Comment by nurvdil** [comment]
i full tested this pr and all tests success ended)

File: zenguard/zenguard.py [hunk 1]
```diff
@@ -135,49 +137,21 @@ def detect(self, detectors: list[Detector], prompt: str):
...
                 f"An error occurred while making the request: {str(e)}"
             ) from e
-        except httpx.HTTPStatusError as e:
-            raise RuntimeError(
-                f"Received an unexpected status code: {response.status_code}\nResponse content: {response.json()}"
-            ) from e
-
-        return response.json()
 
     def detect_async(self, detectors: list[Detector], prompt: str):
```
Comment (baur-krykpayev) [inline zenguard/zenguard.py:150]: @nurvdil now `detect_async` is not really async? we should probably just delete this then and its tests.

**Review by baur-krykpayev — CHANGES_REQUESTED** [review]

File: zenguard/zenguard.py (hunk 1 above)
Comment (nurvdil) [reply zenguard/zenguard.py:150]: Fine. I just decided to do this so that people who used the detect_async method would not break anything

Comment (nurvdil) [reply zenguard/zenguard.py:150]: I just thought that after this pull request I would make the detect_async method really asynchronous using aiohttp and asyncio.gather)

**Comment by baur-krykpayev** [comment]
@baur-krykpayev merge, release new pypi version and test it.

**Review by baur-krykpayev — APPROVED** [review]

**Review by baur-krykpayev — APPROVED** [review]

**Pull request merged**